src/dom/document.ts

```diff
@@ -152,6 +152,14 @@ export class Document extends Node {
   get title(): string {
     return this.querySelector("title")?.textContent || "";
   }
+  set title(value: string) {
+    let titleElement = this.querySelector("title");
+    if (!titleElement) {
```

Comment:
Sorry can you add `&& this.head` in cases where it was removed so that it doesn't throw? Not sure what you want to do with 161 but you basically won't update the title element if no title element nor head element exist